### ISSUE TITLE
Rule Idea: at-mixin-named-arguments and at-function-named-arguments

### ISSUE THREAD

**Issue by diego-codes** [kristerkari/stylelint-scss#181]
## Context
I looked around the project for a mention to this type of rule and didn't find anything, so I apologize if this is a duplicate.

Sass has a really useful feature that allows developers to call mixins and functions with named parameters. It makes otherwise confusing calls with "mystery" arguments more readable and easy to update. I didn't write this [article](http://blog.ricardofilipe.com/post/object-arguments-in-sass), but I agree with this approach. I am advocating in my main project for contributors to adhere to this writing style. I would love it if stylelint-scss checked for this.

## Rules
I believe the two rules should work the same way. The options for both rules could be: `always`, `never`, `always-multiple-arguments`.

### `always`
Bad:
```scss
.foo {
  margin-top: size(10);
}

// or
.foo {
  font-size: em-scale(20, 16);
}

// or 
.foo {
  @include animation(250ms, 100ms, infinite);
} 

// or
.foo {
  @include my-mixin($value: 20, 'bar', $color: #FFF);
}
```

Good:
```scss
.foo {
  font-size: em-scale($size: 20, $base: 16);
}

// or 
.foo {
  @include animation($duration: 250ms, $delay: 100ms, $direction: infinite);
}
```

### `never`
Bad:
```scss
.foo {
  font-size: em-scale($size: 20, $base: 16);
}

// or 
.foo {
  @include animation($duration: 250ms, $delay: 100ms, $direction: infinite);
}

// or
.foo {
  @include my-mixin($value: 20, 'bar', $color: #FFF);
}
```

Good:
```scss
.foo {
  margin-top: size(10);
}

// or
.foo {
  font-size: em-scale(20, 16);
}

// or 
.foo {
  @include animation(250ms, 100ms, infinite);
} 
```

### `always-multiple-arguments`
Bad:
```scss
.foo {
  font-size: em-scale(20, 16);
}

// or 
.foo {
  @include animation(250ms, 100ms, infinite);
}

// or
.foo {
  @include my-mixin($value: 20, 'bar', $color: #FFF);
}
```

Good:
```scss
.foo {
  margin-top: size(10);
}

// or
.foo {
  font-size: em-scale($size: 20, $base: 16);
}

// or 
.foo {
  @include animation($duration: 250ms, $delay: 100ms, $direction: infinite);
}
```

**Comment by kristerkari** [comment]
Thanks! This seems like a really good idea for a rule. :+1:

**Comment by diego-codes** [comment]
Great! I’d be happy to work on a PR for this.

**Comment by kristerkari** [comment]
Nice, please ping me if you need any help :)

**Comment by kristerkari** [comment]
Released in v2.2.0, thanks @diego-codes!

**Comment by diego-codes** [comment]
🎉 awesome! Thanks.
